### DISCUSSION TITLE
14.0 is upon us! - Please check your languages!

### DISCUSSION THREAD

**Discussion by TrueBrain** [OpenTTD/team#497] [Announcements]
Hello @OpenTTD/translators,

As we are nearing a new release, once again I would like to ask you to take a look if your language needs any more translating.

The upcoming release, 14.0, really is a big one. This means for languages that haven't logged in for a while, there really is a lot to do. Languages that were complete in 13.0 are now missing a ~400 strings. And that all to make the game even better!

Sadly, it does mean we created a lot more work for you guys than normally. But I hope you agree with me, especially when looking at the [changelog](https://cdn.openttd.org/openttd-releases/14.0-beta1/changelog.txt) of the upcoming 14.0, it is really worth it.

The release is not for another month; if you can spare some moments between now and then, we would really appreciate it!

As always, thank you so much for your work on bringing OpenTTD alive in over 64 languages. It is truly appreciated!

**Comment by Wuzzy2** [comment]
German language completed!

**Comment by pvillaverde** [comment]
Galician Language also completed!

**Comment by azulcosta** [comment]
Portuguese (european) is completed

A quarta, 14/02/2024, 16:42, Fedello ***@***.***> escreveu:

**Comment by bscargo** [comment]
are working on Danish

**Reply by bscargo** [reply]
Danish is done,

**Comment by SpamixOfficial** [comment]
I'll get started on Swedish today!

**Comment by Afoklala** [comment]
Dutch is ready

-- 
Vriendelijke groeten, friendly greetings, freundliche Grüße,

Jan Willem van Dormolen

Jan Bronnerstraat 116,
2552 WT Den Haag,
Netherlands

Btw-nummer/VAT number: NL001317146B32

tel. *31 - 70 - 3636803
fax  *31 - 84 - 7169254
email ***@***.***
web http://www.xs4all.nl/~dormoljw
skype janwillemvandormolen

And then there's this:
BACHELOR: A man who never makes the same mistake once.

**Reply by Afoklala** [reply]
Dutch is ready again. However, unfortunately, this will be the last work I do on OpenTTD translations. I've moved on from OpenTTD, in fact haven't played it in, like, years. So now is the time to say goodbye. Best of luck with the project to all!

**Comment by LishOwO** [comment]
French is done !

**Comment by bnegrut** [comment]
Romanian is done.

**Comment by LubosKolouch** [comment]
Czech is done.

**Comment by SpamixOfficial** [comment]
Swedish is done

**Comment by metsysma** [comment]
Turkish is done

**Comment by lexuslatvia** [comment]
Latvian done

**Comment by KorneySan** [comment]
Belarusian is done.

Fixed some inconsistency in Russian also.

**Comment by AlphaJack** [comment]
Italian language completed!

**Comment by wensimehrp** [comment]
Simplified Chinese done

**Comment by pasantoro** [comment]
Portuguese (Brazilian) (pt_BR) is done!

Em ter., 6 de fev. de 2024 às 17:35, Patric Stout ***@***.***>
escreveu:

> Hello @OpenTTD/translators
> <https://github.com/orgs/OpenTTD/teams/translators>,
>
> As we are nearing a new release, once again I would like to ask you to
> take a look if your language needs any more translating.
>
> The upcoming release, 14.0, really is a big one. This means for languages
> that haven't logged in for a while, there really is a lot to do. Languages
> that were complete in 13.0 are now missing a ~400 strings. And that all to
> make the game even better!
>
> Sadly, it does mean we created a lot more work for you guys than normally.
> But I hope you agree with me, especially when looking at the changelog
> <https://cdn.openttd.org/openttd-releases/14.0-beta1/changelog.txt> of
> the upcoming 14.0, it is really worth it.
>
> The release is not for another month; if you can spare some moments
> between now and then, we would really appreciate it!
>
> As always, thank you so much for your work on bringing OpenTTD alive in
> over 64 languages. It is truly appreciated!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenTTD/team/discussions/497>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHPAI74ERY5W5M5A6WKNL4DYSKHZLAVCNFSM6AAAAABC4UWLNCVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWGE4DQNJYG4>
> .
> You are receiving this because you are on a team that was mentioned.Message
> ID: ***@***.***>
>


-- 
"Feliz aquele que transfere o que sabe e aprende o que ensina".
(Cora Coralina)
"Pouca leitura acarreta pouco conhecimento,
parco discernimento e baixo poder de análise."
(J. Raul Teixeira)
"Proibir é mostrar-se incapaz de educar".
"Nada muda se eu não mudar".

**Comment by PstasDev** [comment]
A bunch of us made Hungarian done!

**Comment by phreeze83** [comment]
my solowork for Luxembourgish also done ^^

**Comment by ApplePie420** [comment]
I've finished the Slovak translations

**Comment by pasantoro** [comment]
I think that I found some missing characters and some additional ones in the English base language. Who could I send these findings to?

**Reply by krysclarke** [reply]
Anything missing in the base language (EN-GB) needs to be raised as a "bug" on the [OpenTTD github](https://github.com/OpenTTD/OpenTTD/issues).
If you know Git well enough, you could raise a PR with your corrections.

**Comment by Ansbaradigeidfran** [comment]
After a few evenings' work, Welsh (cy_GB) has also been dragged up-to-date for 14.0!

**Comment by StepanIvasyn** [comment]
Ukrainian is done!

**Comment by bscargo** [comment]
15.0-RC2 da_DK is done

**Comment by jnmbk** [comment]
Turkish is done

**Comment by joeax910** [comment]
Swedish is done!

On a side note; I haven't played OpenTTD in years either, because I tend to lose myself in it for longer than I have time for these days, but I'm much too much of a language nerd to not want to translate its strings whenever there are new things to be done...

I don't know you, Jan Willem, but good luck in your future endeavours!

**Comment by bscargo** [comment]
15.0-RC3 dk is done
